pages/_document.js

```diff
@@ -8,6 +8,10 @@ export default class MyDocument extends Document {
           <link rel="preconnect" href="https://fonts.googleapis.com"/>
           <link rel="preconnect" href="https://fonts.gstatic.com" crossorigi="true" />
           <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet"/>
+          <link
```

Comment:
sito irgi nereik. gali tiesiai importint css/js

nes db tu turi irases ta lib locally ir jis vistiek ji sius is cdn